### PR TITLE
Add general code review guide

### PR DESCRIPTION
**Pull request by Ainali**
null

-----
[View rendered criteria/require-review.md](https://github.com/publiccodenet/community-implementation-guide-standard/blob/review-guide/criteria/require-review.md)

### CODE REVIEW

File: criteria/require-review.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ order: 7
 <!-- SPDX-License-Identifier: CC0-1.0 -->
 <!-- written in 2022 by The Foundation for Public Code <info@publiccode.net> -->
 
+This is a short guide that can be useful if you haven't thought about reviewing code from external contributors before: [10 universal steps for open source code review](https://opensource.com/article/22/10/code-review), by Red Hat.
+
```
Comment (ericherman) [inline criteria/require-review.md:10]: This would fit better in "Further reading" rather than above requirements.

**Review by ericherman — CHANGES_REQUESTED** [review]

**Review by ericherman — APPROVED** [review]

File: criteria/require-review.md (hunk 1 above)
Comment (ericherman) [reply criteria/require-review.md:10]: Sorry, my head was in the wrong repo. This is fine.

Comment (Ainali) [reply criteria/require-review.md:10]: Ah yes, it could work as a H2 in the end.

Comment (ericherman) [reply criteria/require-review.md:10]: Could, but I think it is fine as-is.

**Pull request merged**